rdbms/src/test/java/org/spine3/server/storage/jdbc/command/query/Given.java

```diff
@@ -77,7 +77,7 @@ private Given() {
     }
 
     @SuppressWarnings("StaticVariableUsedBeforeInitialization")
```

Comment:
Looks like this suppression makes not much sense, as the `loggerMock` is initialized above (e.g. `loggerMock = null`).